#!/bin/bash

python test-manage.py test provider oauth2 --traceback --failfast


